Plugins/Events/NWScript/nwnx_events.nss

```diff
@@ -1602,6 +1602,26 @@ _______________________________________
     Event Data Tag        | Type   | Notes
     ----------------------|--------|-------
     OBJECT                | object | The Object being used |
+_______________________________________
+    ## Object Open Events (Placeables)
+    - NWNX_ON_OBJECT_OPEN_BEFORE
+    - NWNX_ON_OBJECT_OPEN_AFTER
+
+    `OBJECT_SELF` = The player using the object
+
+    Event Data Tag        | Type   | Notes
+    ----------------------|--------|-------
+    OBJECT                | object | The Object being opened |
```

Comment:
If you want to keep the `ACTION_RESULT` you need to document it here in the event data table